model/historical_test_data.go

```diff
@@ -0,0 +1,69 @@
+package model
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/cedar"
+	"github.com/pkg/errors"
+)
+
+// HistoricalTestData describes aggregated test result data for a given date
+// range.
+type HistoricalTestData struct {
+	Info            HistoricalTestDataInfo
+	NumPass         int       `bson:"num_pass"`
+	NumFail         int       `bson:"num_fail"`
+	AverageDuration float64   `bson:"average_duration"`
+	LastUpdate      time.Time `bson:"last_update"`
+
+	env       cedar.Environment
```

Comment:
I don't think you're using this (yet? ever?)

Reply:
I will because I need to get the configuration to get the test results bucket

Reply:
I broke this into small tickets as a productivity hackl

Reply:
nocie.